docs/json/radarr/cf/x266.json

```
@@ -0,0 +1,24 @@
+{
+  "trash_id": "390455c22a9cac81a738f6cbad705c3c",
+  "name": "x266",
+  "includeCustomFormatWhenRenaming": false,
+  "specifications": [{
+          "name": "x|h266",
+          "implementation": "ReleaseTitleSpecification",
+          "negate": false,
+          "required": true,
+          "fields": {
+              "value": "[xh][ ._-]?266|\\bVVC(\\b|\\d)"
+          }
+      },
+      {
+          "name": "Remux",
```

```suggestion
          "name": "Not Remux",
```
You probably took this from the x265 CF so I guess that one would need changing as well

---

I certainly did! So will change there too.